src/com/sun/ts/lib/deliverable/cts/deploy/DeployTestUtil.java

```diff
@@ -35,16 +35,9 @@
 import javax.enterprise.deploy.spi.*;
 import javax.enterprise.deploy.spi.status.*;
 import javax.enterprise.deploy.spi.exceptions.*;
-import javax.enterprise.deploy.model.*;
 import javax.enterprise.deploy.shared.*;
-import javax.enterprise.deploy.model.*;
-
-import java.io.*;
-import java.util.*;
 
 import javax.enterprise.deploy.shared.factories.*;
```

Comment:
Are you planning to remove the pending occurrences of javax.enterprise.deploy also. Will that be done in another change ?

Reply:
I thought I removed all of the references, I will add an additional commit to remove the missed ones.  

I'll also remove the added wget of  https://repo1.maven.org/maven2/javax/enterprise/deploy/javax.enterprise.deploy-api/1.7/javax.enterprise.deploy-api-1.7.jar in docker/build_jakartaeetck.sh, as we shouldn't reference those classes any more.

Reply:
@alwin-joseph according to jakartaee/src/main/jbake/content/commonappdeploy.adoc, there are additional tests that depend on Jakarta Deployment, however, we removed Jakarta Deployment, so these tests will not work.
```
|Directory Under `com/sun/ts/tests` |Directory Under `com/sun/ts/tests`
With Associated Common Applications
|`ejb/ee/tx/session` |`ejb/ee/tx/txbean`

|`ejb/ee/tx/entity/bmp` |`ejb/ee/tx/txEbean`

|`ejb/ee/tx/entity/cmp` |`ejb/ee/tx/txECMPbean`

|`ejb/ee/tx/entity/pm` |`ejb/ee/tx/txEPMbean`

|`connector/ee/localTx/msginflow` |`common/connector/whitebox`

|`connector/ee/mdb` |`connector/ee/localTx`

|`common/connector/whitebox` |`connector/ee/noTx`

|`common/connector/whitebox` |`connector/ee/xa`

|`common/connector/whitebox` |`connector/ee/connManager`

|`common/connector/whitebox` |`xa/ee`

|`compat13/connector/localTx` |`compat13/connector/whitebox`

|`compat13/connector/noTx` |`compat13/connector/whitebox`

|`compat13/connector/xa` |`compat13/connector/whitebox`

|`interop/tx/session` |`interop/tx/txbean`

|`interop/tx/entity` |`interop/tx/txEbean`

|`interop/tx/webclient` |`interop/tx/txbean`

|`tests/interop/csiv2` |`interop/csiv2/rionly`

|`ejb/ee/pm/ejbql` |`ejb/ee/pm/ejbql/schema`

|`ejb/ee/tx/session/stateful/bm/TxMDBMS_Direct`
|`ejb/ee/tx/session/stateful/bm/TxMDBMSBeans/BeanA`

|`ejb/ee/tx/session/stateful/bm/TxMDBMSBeans/BeanB`
|`ejb/ee/tx/session/stateful/bm/TxMDBMSBeans/BeanC`

|`tests/ejb/ee/tx/session/stateful/bm/TxMDBMS_Indirect`
|`ejb/ee/tx/session/stateful/bm/TxMDBMSBeans/BeanA`

|`ejb/ee/tx/session/stateful/bm/TxMDBMSBeans/BeanB`
|`ejb/ee/tx/session/stateful/bm/TxMDBMSBeans/BeanC`

|`ejb/ee/tx/session/stateful/bm/TxMDBSS_Direct`
|`ejb/ee/tx/session/stateful/bm/TxMDBSSBeans/BeanA`

|`ejb/ee/tx/session/stateful/bm/TxMDBSSBeans/BeanB`
|`tests/ejb/ee/tx/session/stateful/bm/TxMDBSSBeans/BeanC`

|`ejb/ee/tx/session/stateful/bm/TxMDBSS_Indirect`
|`ejb/ee/tx/session/stateful/bm/TxMDBSSBeans/BeanA`

|`ejb/ee/tx/session/stateful/bm/TxMDBSSBeans/BeanB`
|`ejb/ee/tx/session/stateful/bm/TxMDBSSBeans/BeanC`
```

Reply:
I am thinking that the Common Applications Deployment will not work without Jakarta Deployment.  If that is the case, we should remove the Common Applications Deployment tests, docs, support classes.  In theory, we should introduce an alternative to Jakarta Deployment as discussed but we don't have that in place yet, so likely need to remove CAD.

Reply:
Updated plan, to try to get through the build with failures, will remove Common Applications Deployment in a separate pull request.